<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="bwImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="finalImage">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectVideoButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Video</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="videoFilePathText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="frameSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
